src/app/Pages/Dashboard/Home.tsx

```diff
@@ -40,18 +42,25 @@ const Home: React.FunctionComponent<{ title: string }> = ({ title }) => {
             </Title>
           </FlexItem>
           <FlexItem
-            spacer={{ default: 'spacer2xl' }}
-            align={{ default: 'alignRight' }}>
-            <img alt={'Red Hat Enterprise Linux for the Open Hybrid Cloud'} src={logo} style={{ height: 200 }} />
+            spacer={{default: 'spacer2xl'}}
```

Comment:
The space changes are just noise in this commit. I guess it belong to the next one ? (linting fixes)

Reply:
Pushed a new version using prettier to format instead of whatever my IDE was using.